vNext/AISKU/Tests/applicationinsights.e2e.tests.ts

```diff
@@ -365,14 +369,26 @@ export class ApplicationInsightsTests extends TestClass {
                 stepDelay: 5000,
                 steps: [
                     () => {
-                        fetch('https://httpbin.org/status/200', { method: 'GET' });
+                        fetch('https://httpbin.org/status/200', { method: 'GET', headers: { 'header': 'value'} });
                         Assert.ok(true, "fetch monitoring is instrumented");
                     },
                     () => {
                         fetch('https://httpbin.org/status/200', { method: 'GET' });
                         Assert.ok(true, "fetch monitoring is instrumented");
+                    },
+                    () => {
+                        fetch('https://httpbin.org/status/200');
+                        Assert.ok(true, "fetch monitoring is instrumented");
                     }
-                ].concat(this.asserts(2))
+                ]
+                    .concat(this.asserts(3))
+                    .concat(() => {
+                        Assert.ok(this.trackSpy.calledThrice, "trackDependencyDataInternal is called");
+                        Assert.equal("Fetch", this.trackSpy.args[0][0].type, "request is Fetch type");
+                        Assert.equal('value', this.trackSpy.args[0][0].properties.requestHeaders['header'], "fetch request's customer defined request header is stored");
+                        Assert.notEqual(undefined, this.trackSpy.args[0][0].properties.responseHeaders, "fetch request's reponse header is stored");
+                        Assert.equal(undefined, this.trackSpy.args[1][0].properties.requestHeaders['header'], "customer doesn't define a header field");
```

Comment:
check that `properties.requestHeaders` is undefined for `args[2]`

Reply:
ok 👌

Reply:
just curious for this, if the headers are not set, then shouldn't `properties.requestHeaders` not be defined? e.g. this assert will cause the tests to crash